hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/ethereum/EthTxData.java

```diff
@@ -182,7 +113,20 @@ public byte[] encodeTx() {
                     v,
                     r,
                     s);
-            case EIP2930 -> throw new IllegalStateException("EIP2930 txes not supported");
+            case EIP2930 -> RLPEncoder.encodeSequentially(
+                    Integers.toBytes(0x01),
+                    List.of(
+                            chainId,
+                            Integers.toBytes(nonce),
+                            gasPrice,
+                            Integers.toBytes(gasLimit),
+                            to,
+                            Integers.toBytesUnsigned(value),
+                            callData,
+                            List.of(/*accessList*/ ),
+                            Integers.toBytes(recId),
```

Comment:
Here (and at L142 below) why doesn't the actual `v` in this record have to be set to `Integer.toBytes(recId)`?  It's the actual field in the record and here we're encoding something in the `v` location of the rlp thing but the `v` of the record itself doesn't match. ??